.pre-commit-config.yaml

```diff
@@ -7,15 +7,13 @@ repos:
     rev: 21.5b1
     hooks:
     - id: black
-      language_version: python3.8
+      language_version: python
 -   repo: https://gitlab.com/pycqa/flake8
     rev: 3.9.2
     hooks:
     - id: flake8
-# -   repo: local
-#     hooks:
-    # custom hook, rather than mypy hook, gives more control over mypy parameters
-    # - id: mypy-executor
-    #   name: MyPy Executor
-    #   entry: ./mypy.sh
-    #   language: system
\ No newline at end of file
+-   repo: https://github.com/pre-commit/mirrors-mypy
+    rev: v0.910
+    hooks:
+    - id: mypy
+      language_version: python
```

Comment:
Can you explain why you decided to move away from the custom hook?
If we keep this approach, `./mypy.sh` will not be required and should be deleted.

Reply:
I've bumped mypy==v0.910 since it supports configuration in pyproject.toml which I think removes the need for a custom hook